lnwallet/interface.go

```diff
@@ -207,6 +207,11 @@ type WalletController interface {
 	// Stop signals the wallet for shutdown. Shutdown may entail closing
 	// any active sockets, database handles, stopping goroutines, etc.
 	Stop() error
+
+	// BackEnd returns a name for the wallet's backing chain service,
+	// which could be e.g. btcd, bitcoind, neutrino, or another consensus
+	// service.
+	BackEnd() string
```

Comment:
Hmm, idk if this is really necessary. It leaks out implementation details of the underlying `WalletController` implementation.

Reply:
This is mainly used in tests. It's a passthrough driver identification mechanism, since we need to create a `WalletController`. I don't see any other way that this is currently exposed in such a way that the tests have access to it after the instance is created.